wbooks/src/app/screens/Library/components/Book/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Image, View } from 'react-native';
+
+import { IBook } from '@interfaces/book';
+import GenericText from '@app/components/GenericText';
+import { textVariants } from '@app/components/GenericText/constants';
+
+import { styles } from './styles';
+
+function BookImage({ imageUrl }: { imageUrl: string | null }) {
```

Comment:
extract this into a component outside this file. We usually create an inner folder named `components` where we create our inner components definitions. Move it there

it will end something like this:
```
Book (folder)
- index.tsx (file)
- styles.ts (file)
- components (folder)
        - BookImage (folder)
               -  index.tsx (file)
```